LibraryManagementSystem/Forms/ReturnTab.cs

```diff
@@ -45,6 +47,55 @@ private void btnReturn_Click(object sender, EventArgs e)
             con.Close();
 
             txtBookID.Clear();
+            */
+
+            string availText = "Avail";
+
+            if (String.IsNullOrEmpty(txtBookID.Text))
```

Comment:
Add comments to explain what's being checked on if statements and other important details